pubsub/server_test.go

```diff
@@ -52,10 +52,7 @@ func TestServerPublish(t *testing.T) {
 	// Create a new logger for the test
 	logger := logging.NoLog{}
 	// Create a new pubsub server
-	server := New(*dummyAddr, nil, logger, readBufferSize,
-		writeBufferSize, maxPendingMessages, maxMessageSize, writeWait,
-		pongWait, readHeaderTimeout,
-	)
+	server := New(*dummyAddr, nil, logger, NewDefaultServerConfig())
```

Comment:
why not just provide "localhost:8080" here directly? Why is the input a `flag.String`?